cloudzero/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "6f1a61c2-e875-42f7-b8ba-94b191786846",
+  "app_id": "cloudzero",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze your Datadog costs on the CloudZero platform.",
+    "title": "CloudZero",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Category::Cost Management",
+      "Category::Cloud"
```

Comment:
Please also add tags for submitted/queried data types, i.e.:

`Submitted Data Type::Metrics`
`Queried Data Type::Events`